src/components/Resource.tsx

```diff
@@ -8,12 +8,12 @@ import {
   OrderedList,
   Container,
 } from '@chakra-ui/react';
-import Breadcrumbs from './NavComponents/Breadcrumbs';
+// import Breadcrumbs from './NavComponents/Breadcrumbs';
 
 function Resource() {
   return (
     <ChakraProvider>
-      <Breadcrumbs type="resources" />
+      {/* <Breadcrumbs type="resources" /> */}
```

Comment:
Are these breadcrumb components still needed?

Reply:
This resource file isn't being used at all to my knowledge. But in feedback notes was advised to switch the breadcrumbs to a back arrow. Breadcrumbs no longer exists, so I can definitely delete this line now!

Reply:
Okay, great! Nice work.